pkgs/development/tools/misc/texinfo/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib
+, stdenv
+, buildPackages
+, fetchurl
+, perl
+, xz
+, libintl
+, libiconv
+, bash
+, gnulib
+, gawk
+, ncurses
+, procps
+, interactive ? false
+}:
+
+let
+  meta = {
+    description = "The GNU documentation system";
+    homepage = "https://www.gnu.org/software/texinfo/";
+    changelog = "https://git.savannah.gnu.org/cgit/texinfo.git/plain/NEWS";
+    license = lib.licenses.gpl3Plus;
+    platforms = lib.platforms.all;
+    maintainers = with lib.maintainers; [ vrthra oxij ];
```

Comment:
`vrthra` is soft-retired. You can remove themm.

Reply:
✔️ Done